demo/src/app/components/+popover/demos/toggle-manual/toggle-manual.html

```diff
@@ -0,0 +1,5 @@
+<button type="button" class="btn btn-primary"
+        (click)="isOpen = !isOpen"
```

Comment:
what's the point? you're changing `isOpen` but you're not using it as popover's input

use `triggers=""` and `[isOpen]="isOpen"` and toggle `isOpen` by separete button